### DISCUSSION TITLE
[react] Fragment now exported from react/jsx-runtime

### DISCUSSION THREAD

**Discussion by jabrks** [DefinitelyTyped/DefinitelyTyped#69464] [Issues with a @types package]
Since installing the React 18.3.0 types, IntelliSense in VS Code has been suggesting I import `Fragment` from `react/jsx-runtime` rather than `react`. Looking at the diff, it seems this change https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68600 might be related. Is this expected?

[a comment by an automated account is not shown]

**Comment by eps1lon** [comment]
It's expected that this type exist in this entrypoint but not intended that it appears as suggested in auto-imports. Would love to hide types from autocomplete if somebody can figure this out. 

Might have to remove that type ultimately since you're not supposed to manually use this type.
